### DISCUSSION TITLE
Subscribe some topics by topic pattern

### DISCUSSION THREAD

**Discussion by Dengsi** [apache/rocketmq#7297] [General]
比如指定topic pattern为*proxy，订阅所有名称以proxy结尾的topic的消息。Apache Pulsar有这样的功能，不知道Rocketmq是否有此功能，或者通过其他方式实现？

**Comment by onejimmyboy** [comment]
可以尝试发送消息的时候增加tags，消费者消费的时候对tags进行过滤，这样消费者就可以选择性的消费消息。应该能满足你的需求

**Reply by Dengsi** [reply]
感谢回复，但我这边的需求是启动一个Consumer，同时监听名称以proxy结尾的所有topic

**Reply by onejimmyboy** [reply]
看了一下，好像是不支持。支持订阅多个topic，好像不支持模糊匹配
